### ISSUE TITLE
[Feature Request] Easier trigger-spawned AI presences.

### ISSUE THREAD

**Issue by Bubbus** [Bubbus/F3_CA_BUB#113]
**Is your feature request related to a problem? Please describe.**
On Wimmer's "Rising up" mission, some places which were meant to be pre-populated with AI were empty.  Upon speaking with him, it became clear that he intended to Zeus them in during the mission but had to be in too many places at once.  This resulted in a player group entering a major enemy base completely unopposed.  He was aware of trigger spawns and had read the wiki article below, but we both agreed it takes too much effort to implement.

**Describe the solution you'd like**
We have a process to create trigger-spawned AI in the "you're not done yet" wiki: https://github.com/CombinedArmsGaming/CA-Missionmaking-Wiki-By-Bubbus/wiki/Make-spawn-triggers.  This is quite involved, takes a long time to do and is prone to user error.  I suggest making an "all-in-one" function(s) that can be put into an init field of some object, which creates all the needed pieces to spawn an enemy presence mid-mission, automatically.

**Why is this this best way to solve the problem?**
Editor-preplaced AI are a decent choice especially with FW caching, but still have a perf impact and can be drawn away from their intended area via LAMBS etc.  Triggered enemy spawns fill a niche in a Zeused mision - to catch lapses in Zeus concentration and cover for them.

**Additional context**
🤓

**Comment by Joecuronium** [comment]
Feature creep feature creep, lets put in on the docket for 1.6 shall we?

**Comment by dgibso29** [comment]
1.6 for sure.

**Comment by Bubbus** [comment]
I don't think this is even high enough priority to go into 1.6, let's not get ahead of ourselves.  Let's get 1.5 out and then we can decide what goes into 1.6 with less kneejerk involved lol

**Comment by dgibso29** [comment]
It's all a bit fluid ultimately
